util/progress/progressui/display.go

```diff
@@ -248,6 +317,19 @@ func (t *trace) update(s *client.SolveStatus, termWidth int) {
 		}
 		t.byDigest[v.Digest].jobCached = false
 	}
+	for groupID := range groups {
+		group := t.groups[groupID]
+		newVtx := group.mergeSubVtxs()
+		t.triggerVertexEvent(newVtx)
+		if newVtx.Started != nil && group.Started == nil {
```

Comment:
Don't understand this condition. Can't it run multiple times, resulting multiple `append()` calls.

It might have been easier to track all the `client.Vertex` that create a row with the same object. Adding support reference counting for the case where multiple start/complete need to be handled. Without the progress group there is just one reference.

Reply:
> this is the same basic logic that was already in place for vertexes not in a group above.

I see a `prev` check also in the previous logic.

Reply:
The `prev` check is the equivalent of the `group.Started` check here. But now that I understand what we're going for in terms of the start/stop interval timing this will need some updates, which I'm working on.